### PR TITLE
Fix ingress list line heights

### LINKED ISSUE #5902: Ingress list can contain line gaps
Some lines in the `Ingress` list may be empty. Probably multiline table functionality needs to be checked.
ingress list

### PR DESCRIPTION
**Pull request by aleksfront**
Fixes #5902 

https://user-images.githubusercontent.com/9607060/181698918-ace51abb-d57c-4d02-a34e-c54c2e0c9b98.mov

### CODE REVIEW

File: src/renderer/components/virtual-list/virtual-list.tsx [hunk 1]
```diff
@@ -87,7 +87,7 @@ function VirtualListInner<T extends { getId(): string } | string>({
   useEffect(() => {
     try {
       if (prevItems.current.length !== items.length || !isEqual(prevRowHeights.current, rowHeights)) {
-        listRef.current?.resetAfterIndex(0, false);
+        listRef.current?.resetAfterIndex(0);
```
Comment (aleksfront) [inline src/renderer/components/virtual-list/virtual-list.tsx:90]: Removing force render delay attribute allows to update table properly after changing sorting order.

**Review by Nokel81 — DISMISSED** [review]

File: src/renderer/components/+network-ingresses/ingresses.scss [hunk 2]
```diff
@@ -14,8 +18,13 @@
...
-      span:not(:last-of-type) {
-        margin-right: 1em;
+      > div {
+        overflow: hidden;
+        text-overflow: ellipsis;
+      }
+
+      > div + div {
+        margin-top: 6px;
       }
```
Comment (jansav) [inline src/renderer/components/+network-ingresses/ingresses.scss]: These are very easy to break. Add classes.

Comment (ixrock) [reply src/renderer/components/+network-ingresses/ingresses.scss]: `+` it would be easier to refactor when converting to css-modules

**Review by ixrock — DISMISSED** [review]
Why this happens only with Ingress List view? 
It feels like should be fixed at more global level (e.g. table/virtual-list).

Also did you check if it works correctly when table has a scroll with more items? _(didn't notice this in the video)_

Otherwise LGTM.

**Review by Nokel81 — APPROVED** [review]

**Review by ixrock — APPROVED** [review]

**Pull request merged**